content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
+  --env BP_COMPOSER_INSTALL_OPTIONS="--no-dev --prefer-install=auto"
+{{< /code/copyable >}}
+
+To define **global** `composer install` configurations, set the
+`BP_COMPOSER_INSTALL_GLOBAL` environment variable at build-time to a
+space-separated list of global installation options.
+
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_COMPOSER_INSTALL_GLOBAL="friendsofphp/php-cs-fixer squizlabs/php_codesniffer=*"
+{{< /code/copyable >}}
 
-The following options are configurable in the app's `buildpack.yml`
+### Set a custom Composer vendor directory
+To define a custom vendoring location for Composer packages, users can set the
+`COMPOSER_VENDOR_DIR` environment variable at build-time. It is a
+[Composer-native environment
+variable](https://getcomposer.org/doc/03-cli.md#composer-vendor-dir) setting.
+It must be set as a relative path under the application source directory root.
 
 {{< code/copyable >}}
-composer:
-  # this allows you to specify a version constaint for composer
-  # any valid semver constaints (e.g. 1.* and 1.10.*) are also acceptable
-  version: 1.10.x
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER_VENDOR_DIR="vendor"
+{{< /code/copyable >}}
 
-  # a list of command line install options for composer
-  # default: ["--no-dev"]
-  install_options: ["--no-dev"]
+### Set the `composer.json` path
+To define a custom `composer.json` path, users can set [Composer-native
+environment variable `COMPOSER`](https://getcomposer.org/doc/03-cli.md#composer) at build-time.
 
-  # default: vendor
-  vendor_directory: vendor
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER="composer"
+{{< /code/copyable >}}
 
-  # directory where composer.json can be found
-  # default is app root
-  json_path: composer
+### Set Composer authentication
+To set up authentication for Composer, the Composer-native environment variable
+`COMPOSER_AUTH` can be set as a JSON-formatted object containing objects
+defined in the [Composer
+docs](https://getcomposer.org/doc/03-cli.md#composer-auth).
 
-  # if included, will run `composer global` with specified arguments
-  install_global: ["list", "of", "install", "options"]
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER_AUTH='{"http-basic": <some-creds>}'
 {{< /code/copyable >}}
 
-## Select a Web Server
+## Select a web server
 
 The PHP buildpack supports the use of 3 different web servers:
 
- - PHP built-in Web Server
- - Apache HTTP Web Server
+ - PHP Built-in Web Server (default)
+ - Apache HTTPD Web Server
  - Nginx Web Server
 
-You can configure the webserver using `buildpack.yml` as follows:
+You can configure which web server to use by setting the `BP_PHP_SERVER` environment variable at build-time.
+The setting options are `php-server`, `httpd`, `nginx`, with the default value of `php-server` if unset.
```

Comment:
Flagging that this might need to be updated if detection criteria change

Reply:
👍 great call.